assets/runtime/functions

```diff
@@ -819,23 +797,15 @@ initialize_datadir() {
     exit 1
   fi
 
-  # create dotfiles directory
-  mkdir -p ${REDMINE_DOTFILES_DIR}
-  chmod -R u+rw,go+r ${REDMINE_DOTFILES_DIR}
-  chown -R ${REDMINE_USER}: ${REDMINE_DOTFILES_DIR}
-
-  mkdir -p ${REDMINE_DOTFILES_DIR}/.ssh
-  chmod -R u+rw,go-rwx ${REDMINE_DOTFILES_DIR}/.ssh
-  chown -R ${REDMINE_USER}: ${REDMINE_DOTFILES_DIR}/.ssh
-
-  mkdir -p ${REDMINE_DOTFILES_DIR}/.subversion
-  chmod -R u+rw,go+r ${REDMINE_DOTFILES_DIR}/.subversion
-  chown -R ${REDMINE_USER}: ${REDMINE_DOTFILES_DIR}/.subversion
-
-  # create tmp directory
-  mkdir -p ${REDMINE_DATA_DIR}/tmp
-  chmod -R u+rw,go+r ${REDMINE_DATA_DIR}/tmp
-  chown -R ${REDMINE_USER}: ${REDMINE_DATA_DIR}/tmp
+  # fix permissions for relevant folders
+  datafolders="${REDMINE_DATA_DIR} ${REDMINE_PLUGINS_DIR} ${REDMINE_THEMES_DIR} ${REDMINE_ATTACHMENTS_DIR} ${REDMINE_BACKUPS_DIR} ${REDMINE_DOTFILES_DIR} ${REDMINE_DOTFILES_DIR}/.ssh ${REDMINE_DOTFILES_DIR}/.subversion ${REDMINE_DATA_DIR}/tmp"
+  for dir in ${datafolders}; do
```

Comment:
I'd prefer if we use bash arrays just in case the paths ever contain spaces.  None of them do currently but bash always makes me paranoid and shellcheck agrees... 

And putting the list on new lines makes it easier to read. It was a bit hard to see if anything was left out or duplicated.
```
datafolders=(
  "${REDMINE_DATA_DIR}"
  "${REDMINE_PLUGINS_DIR}"
  "${REDMINE_THEMES_DIR}"
  "${REDMINE_ATTACHMENTS_DIR}"
  "${REDMINE_BACKUPS_DIR}"
  "${REDMINE_DOTFILES_DIR}"
  "${REDMINE_DOTFILES_DIR}/.ssh"
  "${REDMINE_DOTFILES_DIR}/.subversion"
  "${REDMINE_DATA_DIR}/tmp"
)
for dir in "${datafolders[@]}"; do
```

Also quote the chmod/chown variable expansions.